known-shortcuts/msteams.ts

```diff
@@ -1,3 +1,5 @@
+import { AppParams } from "~/types";
+
 const app = {
```

Comment:
:bug: **Bug**
The `app` object is being cast to `AppParams` using the `as` keyword. While this is a valid TypeScript pattern, it is generally safer to use type annotations directly when defining the object to ensure that all required properties are present and correctly typed. There is no verification in the provided context that the `app` object actually conforms to the `AppParams` type. This could lead to runtime errors if the `app` object does not match the expected structure of `AppParams`. Consider using TypeScript's type checking to ensure that `app` is correctly typed as `AppParams` from the start, rather than using a type assertion.

```suggestion
const app: AppParams = {
```